recipes/ludoSim/meta.yaml

```diff
@@ -6,10 +6,10 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e04071d847996209c3e0485dd814207619c3c3e3d404894fa531d03622f8c8d5
-  url: https://raw.githubusercontent.com/jaib1/ludoSim/master/LICENSE
-  sha256: 3972dc9744f6499f0f9b2dbf76696f2ae7ad8af9b23dde66d6af86c9dfb36986
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
@isuruf Thank you! Why does the `-` make a difference? Is it just that you need a `-` if you have multiple sources?

Reply:
You need a yaml list.
```
- a: c
- a: d
```
is parsed as `[{a:c}, {a:d}]`

```
a : c
b : d
```
is parsed as `{a: c, b:d}`


```
a : c
a : d
```
is parsed as `{a:d}`